resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -65,15 +68,18 @@ function task_service {
     osdetection
 
     if [ "$OSFAMILY" = "RedHat" ]; then
-        defaultsdir="/etc/sysconfig"
         if [ $MAJREV -lt 7 ]; then
             task install_source_rpm_sysv
         else
             task install_source_rpm_systemd
         fi
     elif [ "$OSFAMILY" = "Debian" ]; then
-        defaultsdir="/etc/default"
-        task install_source_deb
+        sysv_codenames=("squeeze" "wheezy" "lucid" "precise" "trusty")
```

Comment:
I think you'll want to add jessie here as well so source and package remain consistent.